cylc/flow/task_events_mgr.py

```diff
@@ -587,8 +589,18 @@ def process_message(
         # Any message represents activity.
         self.reset_inactivity_timer_func()
 
```

Comment:
Suggest that the `event_time` arg is made non-optional (not a kwarg)

Reply:
Sounds legit - .

Reply:
I've tried it, but actually, not a good idea, because it involved moving a call to `get_current_time_string` in many, many places.

Reply:
Ah my bad I missed the
```py
        if event_time is None:
            event_time = get_current_time_string()
```
above